Budget Manager/task/src/budget/BubbleSort.java

```diff
@@ -0,0 +1,22 @@
+package budget;
+
+public class BubbleSort {
+
+    public BubbleSort() {
```

Comment:
Ligt aan wat je wilt, qua commentaar:
1. Default constructors mag je weglaten.
2. Utility class, dan wil je 'disablen' dat de class aangemaakt kan worden, dan moet je de constructor op private scoping zetten.

Aangezien er 1 methode in de class staat en die is static verwacht ik dat je punt 2 wil.